maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/AbstractBndMavenPlugin.java

```diff
@@ -103,6 +107,9 @@
 	@Parameter(property = "bnd.packagingTypes", defaultValue = PACKAGING_JAR + "," + PACKAGING_WAR)
 	List<String>			packagingTypes;
 
+	@Parameter(property = "skipIfEmpty", defaultValue = "false")
```

Comment:
Is this a good property name? Is `skipIfEmpty` not also used by other plugins? Perhaps we should be `bnd.skipIfEmpty` as the property name?